apps/sms/views/conversation/js/conversation.js

```diff
@@ -725,16 +724,15 @@ var ConversationView = {
 
   // recalling draft for composer only
   // Bug 1010216 might use it for thread drafts too
-  handleDraft: function conv_handleDraft(threadId) {
+  handleDraft: function conv_handleDraft(draftId) {
     // TODO Bug 1010216: remove this.draft
-    var draft = this.draft ||
-      Drafts.get(threadId || Threads.currentId);
+    this.draft = this.draft || Drafts.byDraftId(draftId);
```

Comment:
Maybe we can set `Drafts.byDraftId(draftId)` directly without checking `this.draft` since `this.draft` should not exist before `afterEnterComposer`